index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="background-preview">
+        <header>
+        <div class="logo">
+            <img src="/assets/img/icons/logo.svg" alt="logo" />
+            <p class="logo-text">Travel Portal</p>
+        </div>
+
+        <nav class="navigation">
+            <ul>
+            <li><a href="#">How It Works</a></li>
+            <li><a href="#">Plan Your Trip</a></li>
+            <li><a href="#">Destinations</a></li>
+            <li><a href="#">Travel Stories</a></li>
+            </ul>
+        </nav>
+
+        <div class="button">
+            <button class="btn primary-button">Login</button>
+        </div>
+        </header>
+        <main>
+        <section class="preview">
+            <h1>Explore the beauty of the World</h1>
+            <p>
+            Receive personalized recommendations for countries, hotels, activities
+            and more
+            </p>
+
+            <div class="input">
```

Comment:
Сделай лучше такую структуру: положи `<div class="input">` в тег form. То есть просто поменяй `<div class="input">` и тег form местами. Атрибуты action="#" и method="post" у формы можно пока убрать. 
```suggestion
      <form>
         <div class="input">
                <input type="search" placeholder="What would you like to do?" />
                <button class="btn input-button">Start Planning</button>
         </div>
       </form>
```